runtime/src/main/java/com/tns/AndroidJsV8Inspector.java

```diff
@@ -93,6 +99,75 @@ private static String getInspectorMessage(Object connection) {
         return ((JsV8InspectorWebSocket) connection).getInspectorMessage();
     }
 
+    @RuntimeCallable
+    public static Pair<String, String>[] getPageResources() {
+        // necessary to align the data dir returned by context (emulator) and that used by the v8 inspector
+        if (applicationDir.startsWith("/data/user/0/")) {
+            applicationDir = applicationDir.replaceFirst("/data/user/0/", "/data/data/");
+        }
+
+        String dataDir = applicationDir;
+        File rootFilesDir = new File(dataDir, "app");
```

Comment:
ts_helpers.js is parsed by the V8, and thus - is debuggable.

livesync on the other hand doesn't appear in the sources tab and will be hidden with the current implementation of the page agent. But then again I haven't seen it being used, and there isn't much to debug in the source anyway.